ingestion/functions/parsing/riograndedosul/riograndedosul.py

```diff
@@ -0,0 +1,246 @@
+    if raw_ethnicity == "Preta":
+        return "Black"
+    elif raw_ethnicity == "Parda":
+        return "Mixed"
+    elif raw_ethnicity == "Amarela":
+        return "Asian"
+    elif raw_ethnicity == "Branca":
+        return "White"
+    elif raw_ethnicity == "Indigena":
+        return "Indigenous"
+
+
+def convert_demographics(gender: str, age: str, ethnicity: str):
+    if not any((gender, age, ethnicity)):
+        return None
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        if age == "80 e mais":
+            demo["ageRange"] = {
+                "start": 80,
+                "end": 120
+            }
+        elif age == "<1":
+            demo["ageRange"] = {
+                "start": 0,
+                "end": 0
+            }
+        else:
+            age_range = age.split(' a ')
+            demo["ageRange"] = {
+                "start": float(age_range[0]),
+                "end": float(age_range[1])
+            }
+    if ethnicity != "NAO INFORMADO":
+        demo["ethnicity"] = convert_ethnicity(ethnicity.title())
+    return demo
+
+
+def convert_notes(raw_commorbidities: str, raw_notes_neighbourhood: str, raw_notes_indigenousEthnicity: str):
+    raw_notes = []
+    if "Portador de doenças cromossômicas ou estado de fragilidade imunológica" in raw_commorbidities:
+        raw_notes.append(
+            "primary immunodeficiency disease or chromosomal disease")
+    if "Imunossupressão" in raw_commorbidities:
+        raw_notes.append("Patient with immunosupression")
+    if "Imunodeficiênci" in raw_commorbidities:
+        raw_notes.append("Patient with immunodeficiency")
+    if raw_notes_neighbourhood:
+        raw_notes.append("Neighbourhood: " + raw_notes_neighbourhood.title())
+    if raw_notes_indigenousEthnicity != "NAO ENCONTRADO":
+        raw_notes.append("Indigenous ethnicity: " +
+                         raw_notes_indigenousEthnicity.title())
+    notes = (', ').join(raw_notes)
+    if notes == '':
+        return None
+    return notes
+
+
+def parse_cases(raw_data_file: str, source_id: str, source_url: str):
+    """
+    Parses G.h-format case data from raw API data.
+    """
+    with open(raw_data_file, "r") as f:
+        reader = csv.reader(f, delimiter=';')
+        next(reader)  # Skip the header.
+        for row in reader:
+            try:
+                case = {
+                    "caseReference": {
+                        "sourceId": source_id,
+                        "sourceUrl": source_url
+                    },
+                    "location": {"query": ", ".join([row[_COUNTY], "Rio Grande do Sul", "Brazil"])},
+                    "events": convert_events(
+                        row[_DATE_CONFIRMED],
+                        row[_DATE_SYMPTOMS],
+                        row[_HOSPITALIZED],
+                        row[_DATE_DEATH]),
+                    "demographics": convert_demographics(
```

Comment:
You may want to include the information included in the 'PROFISSIONAL_SAUDE' entry, namely whether the patient is a Healthcare worker, in 'occupation'.